ledger/eval.go

```diff
@@ -467,25 +467,14 @@ func (eval *BlockEvaluator) transaction(txn transactions.SignedTxn, ad transacti
 			return TransactionInLedgerError{txn.ID()}
 		}
 
-		// Well-formed on its own?
-		err = txn.Txn.WellFormed(spec, eval.proto)
```

Comment:
I deleted this because `WellFormed` is immediately called by `verify.TxnPool` below. Can someone please double check this for me since it's... pretty important

Reply:
can't remove because the transaction cache might become invalid over a protocol upgrade

Reply:
Yup. Txn and TxnPool both call WellFormed immediately before inline or pool based execution of stxnVerifyCore. I'm guessing that's out of some idea to catch simple things before going through the overhead of pool enqueue/dequeue.